### DISCUSSION TITLE
FTPS in Batocera Linux no longer works.

### DISCUSSION THREAD

**Discussion by michaelrasbury** [iterate-ch/cyberduck#15964] [Q&A]
Until a recent version of CyberDuck, I could connect via FTPS (SSH) to my Batocera linux machine.  I can still do this on other machines running much earlier versions of CyberDuck and on the machine in question can SSH in through terminal successfully.  This was working fine in CyberDuck on this machine until a couple of weeks ago.  Did something change?
[cyberduck.log](https://github.com/iterate-ch/cyberduck/files/15376731/cyberduck.log)

**Comment by michaelrasbury** [comment]
For some unknown reason SFTP became available after a few days. I’m not sure why it started working again…

**Comment by michaelrasbury** [comment]
I spoke too soon. I can't connect again. This happened right after I swapped my linux hard drive out for a direct copy of it on a newer drive. I can reach the server through Mac terminal using ssh just fine with the same credentials.

**Comment by tatze96** [comment]
Based on the logging output, you are running into connection timeouts. You can try to work around the timeout issues by increasing the *Timeout for opening connections* within the Cyberduck preferences *Connection* tab.
